python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -196,8 +197,15 @@ def _init_splited_vars(self, split_method):
         self._update_dist_lookup_table_vars(param_list, grad_list,
                                             self.params_grads)
 
-        grad_blocks = split_variable(grad_list, len(self.pserver_endpoints))
-        param_blocks = split_variable(param_list, len(self.pserver_endpoints))
+        if align_var_to_block:
```

Comment:
That's smart! `align_var_to_block` this argument may be changed to some name easier to understand? like `do_slice`?

Reply:
Sounds like `is_slice_var` is to say is this variable sliced?

Reply:
How about ```do_var_slice_up```

Reply:
I changed the variables name and the logic of _init_splited_vars, please review it